docs/sphinx_api_docs_source/build_sphinx_api_docs.py

```diff
@@ -601,7 +601,7 @@ def _clean_up_code_blocks(self, doc: str) -> str:
         Also quotes use a different quote character. This method cleans up
         these items so that the code block is rendered appropriately.
         """
-        doc = doc.replace("&lt;", "<").replace("&gt;", ">")
+        doc = doc.replace("&lt;", "<").replace("&gt;", ">").replace("&amp;42;", "*")
```

Comment:
Why did we get this value?

Reply:
This is a result of the `sphinx-build` process, that escapes HTML special characters displaying them like that to avoid breaking the HTML structure. As the affected part of the docs is generated directly from the code signature (not docstrings) replacing them here seems to be the easiest way to fix, let me know if you think otherwise!

Reply:
The original character is an asterisk, you can check that by commenting the code that makes the signatures render as code blocks in `utils.py`:
![image](https://github.com/user-attachments/assets/78cc80c3-02de-4b8b-b0e9-6bfec29fe093)
![image](https://github.com/user-attachments/assets/be34e83c-4a9c-4113-8578-85f9d5e62107)